src/PactNet/Verifier/PactVerifier.cs

```diff
@@ -34,7 +35,7 @@ public PactVerifier(PactVerifierConfig config) : this(new Dictionary<string, str
         /// </summary>
         /// <param name="verifierArgs">Pact verifier args</param>
         /// <param name="config">Pact verifier config</param>
-        internal PactVerifier(IDictionary<string, string> verifierArgs, PactVerifierConfig config)
+        public PactVerifier(IDictionary<string, string> verifierArgs, PactVerifierConfig config)
         {
```

Comment:
This constructor is deliberately internal only. Users of the library shouldn't be able to construct it.

Reply:
@adamrodger  done